tools/apollo/create_or_update_organism.py

```diff
@@ -36,7 +36,7 @@ def IsOrgCNSuffixEnabled():
     if 'GALAXY_APOLLO_ORG_SUFFIX' not in os.environ:
         return False
     value = os.environ['GALAXY_APOLLO_ORG_SUFFIX']
-    if value.lower() in ('true', 't', '1'):
+    if value.lower() in ('id', 'email'):
```

Comment:
you cast to lower here, but not in the later check. Maybe just `return "id"` and `return "email"` here and use the output of this to check which branch below?